R/copernicus_article.R

```diff
@@ -12,21 +12,21 @@
 #'
 #' An number of required and optional manuscript sections, e.g. \code{acknowledgements}, \code{competinginterests}, or \code{authorcontribution}, must be declared using the respective properties of the R Markdown header - see skeleton file.
 #'
-#' \strong{Version:} Based on copernicus_package.zip in the version 5.3, 18 February 2019, using \code{copernicus.cls} in version 8.82.
+#' \strong{Version:} Based on \code{copernicus_package.zip} in the version 6.2, 15 January 2021, using \code{copernicus.cls} in version 9.25.
 #'
-#' \strong{Copernicus journal abbreviations:} You can use the function \code{copernicus_journal_abbreviations()} to get the journal abbreviation for all journals supported by the copernicus article template.
+#' \strong{Copernicus journal abbreviations:} You can use the function \code{copernicus_journal_abbreviations()} to get the journal abbreviation for all journals supported by the Copernicus article template.
 #'
 #' \strong{Important note:} The online guidelines by Copernicus are the official resource.
 #' Copernicus is not responsible for the community contributions made to support the template in this package.
-#' Copenicus converts all typeset TeX files into XML, the expressions and markups have to be highly standardized.
+#' Copernicus converts all typeset TeX files into XML, the expressions and markups have to be highly standardized.
 #' Therefore, please keep the following in mind:
 #'
 #' \itemize{
 #'   \item Please provide only one figure file for figures with several panels, and please do not use \code{\\subfloat} or similar commands.
 #'   \item Please use only commands in which words, numbers, etc. are within braces (e.g. \code{\\textrm{TEXT}} instead of \code{{\\rm TEXT}}).
 #'   \item For algorithms, please use the syntax given in template.tex or provide your algorithm as a figure.
 #'   \item Please do not define new commands.
-#'   \item The most commonly used packages (\code{\\usepackage{}}) are integrated in the copernicus.cls. Some other packages often used by the community are defined in template.tex. Please do not insert additional ones in your *.tex file.
+#'   \item The most commonly used packages (\code{\\usepackage{}}) are integrated in the \code{copernicus.cls}. Some other packages often used by the community are defined in template.tex. Please do not insert additional ones in your *.tex file.
 #'   \item Spaces in labels (\code{\\label{}}) are not allowed; please make sure that no label name is assigned more than once.
 #'   \item Please do not use \code{\\paragraph{}}; only \code{\\subsubsection{}} is allowed.
 #'   \item It is not possible to add tables in colour.
```

Comment:
Maybe a rule can be added here about the highlighting stuff: 

* Do not forget to use `highlight = NULL` for editor review. It has been set to NULL to not forget it. It can be set to other value during preprint step for example for conveniance.

It is what I understood.

Reply:
This is an important point, thank you. I added more information to the documentation as requested.